ProjectBank/Client/Pages/MyProjects.razor

```diff
@@ -0,0 +1,21 @@
+@page "/user"
+
+<PageTitle>My Projects</PageTitle>
+
+<h1>My Projects</h1>
+
+<ul>
+    @foreach (var project in user.Projects)
+    {
+        <li>@project.Title</li>
+    }
+</ul>
+
+@code {
+
+    User user = new User("Thore", "thore@itu.dk", "I<3Rico")
```

Comment:
As much as i absolutely adore this <333 maybe we should keep it inline with the rest and use generic names: https://da.wikipedia.org/wiki/Alice_og_Bob